Octokit/Models/Request/RepositoryHooksPingRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Octokit
+{
+    public class RepositoryHooksPingRequest
```

Comment:
As above, if we've got an empty class here I think we can just do away with it for now.